CGMBLEKitUI/TransmitterSettingsViewController.swift

```diff
@@ -158,10 +160,40 @@ class TransmitterSettingsViewController: UITableViewController {
         formatter.doesRelativeDateFormatting = true
         return formatter
     }()
-
+    
+    private lazy var sensorExpFullFormatter: DateFormatter = {
+        let formatter = DateFormatter()
+        formatter.dateFormat = "E, MMM d 'at' h:mm a"
```

Comment:
This is non-localized.  Maybe just use `dateFormatter` above.

Reply:
I use this formatter to display the day of the week, which dateFormatter does not. I've modified this formatter to use ".full" dateStyle and ".short" timeStyle. If we could find a way to localize this though, A briefer, abbreviated date might look a little cleaner.